examples/analytics-vendors.amp.html

```diff
@@ -95,6 +95,7 @@
       <option>quantcast</option>
       <option>rakam</option>
       <option>reppublika</option>
+      <option>kenshoo</option>
```

Comment:
Should be in alphabetical order 😄

Reply:
Fixed. totally missed that :see_no_evil: thanks!